electron_app/src/electron-main.js

```diff
@@ -86,8 +86,16 @@ const store = new Store({ name: "electron-config" });
 
 let mainWindow = null;
 global.appQuitting = false;
-global.minimizeToTray = store.get('minimizeToTray', true);
+global.showTrayIcon = store.get("showTrayIcon", true);
+global.minimizeToTray = global.showTrayIcon && store.get('minimizeToTray', true);
 
+// It's important to call `path.join` so we don't end up with the packaged asar in the final path.
+const iconFile = `riot.${process.platform === 'win32' ? 'ico' : 'png'}`;
+const iconPath = path.join(__dirname, "..", "..", "img", iconFile);
```

Comment:
I am unsure how this path works and indeed locally it does not work yet the final builds seem fine

Reply:
It depends on a packaging step to copy the images outside the asar:

https://github.com/vector-im/riot-web/blob/ddf733a7cc59297aa00d780c9dfd09130c07821d/package.json#L162-L169